docker/ftest-docker-machine-network/src/test/java/org/arquillian/cube/servlet/PingPongTest.java

```diff
@@ -107,6 +107,7 @@ public void container_should_have_static_ip_for_app_net_network() throws Interru
                 .exec(new ExecStartResultCallback(outputStream, errorStream)).awaitCompletion();
 
         assertThat(outputStream.toString()).contains("inet addr:172.16.238.10", "inet6 addr: fe80::42:acff:fe10:ee0a/64");
-
+        outputStream.close();
+        errorStream.close();
     }
```

Comment:
I believe we should use `try-with-resources` instead. Or at least try-catch-finally idiom